lib/app/modules/LoginView/controllers/login_view_controller.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:get/get.dart';
+
+class LoginViewController extends GetxController {
+  //TODO: Implement LoginViewController
+
+  final count = 0.obs;
+
+
```

Comment:
add files that were created automatically by get_cli , i.e dispose method, onReady , onClose method, don't remove this methods as they are helpful to prevent memory leakges